profiles/serializers.py

```diff
@@ -160,7 +160,7 @@ class ProfileBaseSerializer(ModelSerializer):
 
     def get_username(self, obj):
         """Getter for the username field"""
-        return get_social_username(obj.user)
+        return get_social_username(obj.user) if hasattr(obj, 'user') else None
```

Comment:
I believe I ran into a case where it doesn't but I'll see I I can still reproduce it.